app/run-test-debug.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
Can you help me understand the point of this? Seems like a duplicate of the existing `run-test.sh` script, with minor tweaks.

Thanks.

Reply:
It outputs all information necessary to check why a test fails, and debug it without having to refer to lots of different files.

I'll take this out of the PR. A better soluttion is a "debug" or "verbose" parameter in the run test.sh script.

Reply:
When running tests regularly, it's very helpful to have all relevant output immediately avaialble without having to examine different output files. I agree that this is not an elegant solution I'll take it out and make a "verbose" flag for the run-test script.